### PR TITLE
Jenkins jobs and plugins configurations for release-2.3.0

### PR DESCRIPTION
**Pull request by SMYALTAMASH**

### CODE REVIEW

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Zookeeper/config.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Zookeeper/config.xml]: @SMY-ALTAMASH Same comment

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Search/config.xml [hunk 2]
```diff
@@ -95,7 +95,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Search/config.xml]: @SMY-ALTAMASH Why is this pointing to devops repo? All KP should be in KP

Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Provision/jobs/dev/jobs/KnowledgePlatform/jobs/Search/config.xml]: good catch will change it

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/health/config.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/health/config.xml]: Is this required?

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/redis-restore/config.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      <limitOneJobWithMatchingParams>false</limitOneJobWithMatchingParams>
+      <paramsToUseForLimit></paramsToUseForLimit>
+    </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
+  </properties>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.67">
+    <scm class="hudson.plugins.git.GitSCM" plugin="git@3.9.3">
+      <configVersion>2</configVersion>
+      <userRemoteConfigs>
+        <hudson.plugins.git.UserRemoteConfig>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/redis-restore/config.xml]: change to data pipeline

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Bootstrap/config.xml [hunk 5]
```diff
@@ -90,12 +90,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
...
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
         </hudson.plugins.git.UserRemoteConfig>
       </userRemoteConfigs>
       <branches>
         <hudson.plugins.git.BranchSpec>
-          <name>${branch_or_tag}</name>
+          <name>refs/tags/tag-for-backups</name>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Bootstrap/config.xml]: change to branch or tag.

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Bootstrap/config.xml [hunk 6]
```diff
@@ -90,12 +90,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Bootstrap/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CassandraBackup/config.xml [hunk 7]
```diff
@@ -101,12 +102,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
...
     </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
   </properties>
-  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.66">
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.67">
     <scm class="hudson.plugins.git.GitSCM" plugin="git@3.9.3">
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CassandraBackup/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CassandraRestore/config.xml [hunk 8]
```diff
@@ -82,22 +82,32 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
...
     </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
   </properties>
-  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.66">
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.67">
     <scm class="hudson.plugins.git.GitSCM" plugin="git@3.9.3">
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CassandraRestore/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CreateUser/config.xml [hunk 9]
```diff
@@ -117,7 +117,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       </userRemoteConfigs>
       <branches>
         <hudson.plugins.git.BranchSpec>
-          <name>${branch_or_tag}</name>
+          <name>refs/tags/tag-for-backups</name>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/CreateUser/config.xml]: change to b or t

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/ElasticsearchBackup/config.xml [hunk 10]
```diff
@@ -94,7 +94,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/ElasticsearchBackup/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/ElasticsearchRestore/config.xml [hunk 11]
```diff
@@ -87,19 +87,21 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
...
     </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
   </properties>
-  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.66">
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.67">
     <scm class="hudson.plugins.git.GitSCM" plugin="git@3.9.3">
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/ElasticsearchRestore/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/KeyRotation/config.xml [hunk 12]
```diff
@@ -122,7 +122,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       </userRemoteConfigs>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/KeyRotation/config.xml]: delete this

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Neo4jBackup/config.xml [hunk 13]
```diff
@@ -94,7 +94,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Neo4jBackup/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Neo4jRestore/config.xml [hunk 14]
```diff
@@ -94,12 +94,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
...
     </hudson.plugins.throttleconcurrents.ThrottleJobProperty>
   </properties>
-  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.66">
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.67">
     <scm class="hudson.plugins.git.GitSCM" plugin="git@3.9.3">
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/Neo4jRestore/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/RedisBackup/config.xml [hunk 15]
```diff
@@ -94,7 +94,7 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
       <configVersion>2</configVersion>
       <userRemoteConfigs>
         <hudson.plugins.git.UserRemoteConfig>
-          <url>https://github.com/project-sunbird/sunbird-learning-platform.git</url>
+          <url>https://github.com/project-sunbird/sunbird-devops.git</url>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/RedisBackup/config.xml]: change to lp

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/health/config.xml [hunk 16]
```diff
@@ -0,0 +1,43 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/KnowledgePlatform/jobs/health/config.xml]: required?

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogElasticSearch/config.xml [hunk 17]
```diff
@@ -25,15 +25,15 @@
           <randomName>choice-parameter-196488765575840</randomName>
           <visibleItemCount>1</visibleItemCount>
           <script class="org.biouno.unochoice.model.GroovyScript">
-            <secureScript plugin="script-security@1.57">
+            <secureScript plugin="script-security@1.58">
               <script>if (override_private_branch.equals(&quot;true&quot;)) {
-return &quot;&quot;&quot;&lt;input name=&quot;value&quot; value=&quot;${private_repo_branch}&quot; class=&quot;setting-input&quot;  type=&quot;text&quot;&gt;&lt;br&gt; &lt;font color=dimgray size=2&gt;&lt;b&gt;Change this value to checkout a different branch from private repository.&lt;/b&gt;&lt;/font&gt;&quot;&quot;&quot;
+return &quot;&quot;&quot;&lt;input name=&quot;value&quot; value=&quot;db_restore&quot; class=&quot;setting-input&quot;  type=&quot;text&quot;&gt;&lt;br&gt; &lt;font color=dimgray size=2&gt;&lt;b&gt;Change this value to checkout a different branch from private repository.&lt;/b&gt;&lt;/font&gt;&quot;&quot;&quot;
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogElasticSearch/config.xml]: change to PRB

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogEsUpgrade5xx/config.xml [hunk 18]
```diff
@@ -0,0 +1,115 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogEsUpgrade5xx/config.xml]: is this working? if not delete

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogEsUpgrade6xx/config.xml [hunk 19]
```diff
@@ -0,0 +1,115 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/LogEsUpgrade6xx/config.xml]: if not working, delete

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/keycloak-test/config.xml [hunk 20]
```diff
@@ -0,0 +1,114 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/keycloak-test/config.xml]: delete

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/user-keycloak/config.xml [hunk 21]
```diff
@@ -0,0 +1,115 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<flow-definition plugin="workflow-job@2.32">
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/user-keycloak/config.xml]: required?

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Kafka/config.xml [hunk 22]
```diff
@@ -77,6 +77,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
+        <hudson.model.StringParameterDefinition>
+          <name>remote</name>
+          <description></description>
+          <defaultValue>processing-cluster-kafka</defaultValue>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Kafka/config.xml]: what is this default? @Kaali09

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml [hunk 23]
```diff
@@ -77,30 +77,22 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
-        <hudson.model.StringParameterDefinition>
-          <name>hosts</name>
-          <description>&lt;font color=dimgray size=2&gt;&lt;b&gt;Specify IP or group to limit the run on specific IP or host&lt;/b&gt;&lt;/font&gt;</description>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml]: Is this field not required? @Kaali09

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Zookeeper/config.xml [hunk 24]
```diff
@@ -76,6 +76,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
+        <hudson.model.StringParameterDefinition>
+          <name>remote</name>
+          <description></description>
+          <defaultValue>processing-cluster-zookeepers</defaultValue>
```
Comment (keshavprasadms) [inline deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Zookeeper/config.xml]: whats is this default? @Kaali09

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/user-keycloak/config.xml (hunk 21 above)
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Provision/jobs/dev/jobs/Core/jobs/user-keycloak/config.xml]: yes

File: deploy/jenkins/jobs/Build/jobs/KnowledgePlatform/jobs/LPFuntionalTest/config.xml [hunk 25]
```diff
@@ -0,0 +1,121 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
[the remark this answers is not in the document]
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Build/jobs/KnowledgePlatform/jobs/LPFuntionalTest/config.xml]: this is required

[the remark this answers is not in the document]
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Build/jobs/KnowledgePlatform/jobs/LPFuntionalTest/config.xml]: thsi file is required

File: deploy/jenkins/jobs/Deploy/jobs/dev/jobs/Core/jobs/KeycloakCustom/config.xml [hunk 26]
```diff
@@ -0,0 +1,115 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
[the remark this answers is not in the document]
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Deploy/jobs/dev/jobs/Core/jobs/KeycloakCustom/config.xml]: its required

File: deploy/jenkins/jobs/Deploy/jobs/dev/jobs/Core/jobs/user-Keycloak/config.xml [hunk 27]
```diff
@@ -0,0 +1,220 @@
+<?xml version='1.1' encoding='UTF-8'?>
```
[the remark this answers is not in the document]
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/Deploy/jobs/dev/jobs/Core/jobs/user-Keycloak/config.xml]: yes

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/health/config.xml (hunk 3 above)
Comment (SMYALTAMASH) [reply deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/DataPipeline/jobs/health/config.xml]: yes

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Kafka/config.xml (hunk 22 above)
Comment (Kaali09) [reply deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Kafka/config.xml]: We have variablised the hostname in the kafka provision playbook.yml. So default is processing-cluster-kafka.

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Zookeeper/config.xml (hunk 24 above)
Comment (Kaali09) [reply deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Zookeeper/config.xml]: We have variablised the hostname in the zookeeper provision playbook.yml. So default is processing-cluster-kafka.

File: deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml (hunk 23 above)
Comment (Kaali09) [reply deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml]: This  is not required.

File: deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/Core/jobs/ESMapping/config.xml [hunk 28]
```diff
@@ -13,10 +13,20 @@
...
-    <com.sonyericsson.rebuild.RebuildSettings plugin="rebuild@1.29">
+    <com.sonyericsson.rebuild.RebuildSettings plugin="rebuild@1.30">
       <autoRebuild>false</autoRebuild>
       <rebuildDisabled>false</rebuildDisabled>
     </com.sonyericsson.rebuild.RebuildSettings>
+    <hudson.model.ParametersDefinitionProperty>
+      <parameterDefinitions>
+        <hudson.model.StringParameterDefinition>
+          <name>branch_name</name>
+          <description>Specify the branch name to clone the sunbird-utils repo where the index json mapping files are present.</description>
```
[the remark this answers is not in the document]
Comment (Kaali09) [reply deploy/jenkins/jobs/OpsAdministration/jobs/dev/jobs/Core/jobs/ESMapping/config.xml]: added.

File: deploy/jenkins/jobs/Deploy/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml [hunk 29]
```diff
@@ -162,6 +162,36 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
+        <org.biouno.unochoice.CascadeChoiceParameter plugin="uno-choice@2.1">
+          <name>job_names_to_deploy</name>
+          <description>&lt;font color=green size=2&gt;&lt;b&gt;Choose the jobs to kill and multi selection also possible&lt;/b&gt;&lt;/font&gt;</description>
```
[the remark this answers is not in the document]
Comment (Kaali09) [reply deploy/jenkins/jobs/Deploy/jobs/dev/jobs/DataPipeline/jobs/Yarn/config.xml]: @keshavprasadms The description is improved.

**Pull request merged**